datahub-frontend/app/controllers/SsoCallbackController.java

```diff
@@ -126,7 +133,18 @@ public Result perform(
   }
 
   private boolean shouldHandleCallback(final String protocol) {
-    return _ssoManager.isSsoEnabled()
-        && _ssoManager.getSsoProvider().protocol().getCommonName().equals(protocol);
+    if (!_ssoManager.isSsoEnabled()) {
+      return false;
+    }
+    updateConfig();
```

Comment:
My original impression was that this was doing a refresh of the SSO provider by fetching configs from the backend... Is that no longer required?

Reply:
That happens in a different part, see SSOManager.getDynamicSettings. That makes a call out to the AuthServiceController getting the settings from GMS.

Reply:
Do you mean the OIDC backend or GMS? There is separate logic to pull the configuration from GMS in SSOManager.getDynamicSettings which calls out to the AuthServiceController and the OIDC backend update happens in a separate flow controlled by AuthenticationController